sdk/keyvault/azure-keyvault-secrets/samples/list_operations.py

```diff
@@ -19,7 +19,7 @@
 #
 # ----------------------------------------------------------------------------------------------------------
 # Sample - demonstrates the basic list operations on a vault(secret) resource for Azure Key Vault.
-# The vault has to be soft-delete enabled to perform one of the following operations. [Azure Key Vault soft delete]
+# The vault has to be soft-delete enabled to perform one of the following operations. [Azure Key Vault soft-delete]
```

Comment:
Better to replace the hopeless, broken markdown with e.g. "See https://... for more information about soft-delete".

Reply:
replaced the broken markdown links